pandas/core/config.py

```diff
@@ -803,3 +803,4 @@ def inner(x):
 is_str = is_type_factory(str)
 is_unicode = is_type_factory(compat.text_type)
 is_text = is_instance_factory((str, bytes))
+is_callable = callable
```

Comment:
@jreback: This doesn't work because `callable` doesn't raise a ValueError. I could overcome this by: 

```
def is_callable(obj):
    """

    Parameters
    ----------
    `obj` - the object to be checked

    Returns
    -------
    validator - returns True if object is callable, and
        raises ValueError otherwise.

    """
    if not callable(obj):
        raise ValueError("Value must be an callable")
    return True
```

Does that make sense?